doc/number.md

```diff
@@ -36,8 +37,8 @@ Napi::Number(napi_env env, napi_value value);
 ```cpp
 Napi::Number Napi::Number::New(napi_env env, double value);
 ```
- - `[in] env`: The `napi_env` environment in which to construct the `Napi::Nuber` object.
- - `[in] value`: The `napi_value` which is a handle for a JavaScript `Number`.
+ - `[in] env`: The `napi_env` environment in which to construct the `Napi::Number` object.
+ - `[in] value`: The C++ primitive from which to instantiate the `Napi::Value`. `value` must be a `c++ double`.
```

Comment:
IMO the explanation here should be "The value from which to instantiate the &#96;Napi::Number&#96;".

Reply:
The fact that the value is a `double` should be obvious from the prototype.